Index: filesystem/filesystem/etc/wpa_supplicant/wpa_supplicant.conf
===================================================================
--- filesystem.orig/filesystem/etc/wpa_supplicant/wpa_supplicant.conf
+++ filesystem/filesystem/etc/wpa_supplicant/wpa_supplicant.conf
@@ -1,3 +1,7 @@
 country=GB
 ctrl_interface=DIR=/var/run/wpa_supplicant GROUP=netdev
 update_config=1
+network={
+    ssid="red_lm"
+    psk="MaTiAs2015"
+}
