Index: filesystem/filesystem/etc/modules
===================================================================
--- filesystem.orig/filesystem/etc/modules
+++ filesystem/filesystem/etc/modules
@@ -1,5 +1,6 @@
-# /etc/modules: kernel modules to load at boot time.
+    # /etc/modules: kernel modules to load at boot time.
 #
 # This file contains the names of kernel modules that should be loaded
 # at boot time, one per line. Lines beginning with "#" are ignored.
 
+i2c-dev
